unblob/processing.py

```diff
@@ -146,6 +149,18 @@ def __init__(
     def process(self):
         logger.debug("Processing file", path=self.task.path, size=self.size)
 
+        detect = magic.detect_from_filename(self.task.path)
+        skip = any(
+            [detect.name.startswith(pattern) for pattern in self.config.skip_magic]
+        )
+        if skip:
+            logger.debug(
+                "Skip file from processing based on file-magic",
+                magic=detect.name,
+                path=self.task.path,
+            )
+            return
+
```

Comment:
For me it'd make sense to extract this into a `_should_skip` method inside `Processor` and call from the `if..elif..` chain in `process_task`. That way all the logic determining if we want to process a file would be in the same place.

Reply:
done